### ISSUE TITLE
runtime.native.System.Data.SqlClient.sni in release/1.1.0 references RTM 1.0 packages

### ISSUE THREAD

**Issue by ericstj** [dotnet/corefx#12195]
Currently the package depends on the following in 1.1:
```xml
      <dependency id="runtime.win10-arm64.runtime.native.System.Data.SqlClient.sni" version="4.3.0-beta-24514-00" />
      <dependency id="runtime.win7-x64.runtime.native.System.Data.SqlClient.sni" version="4.0.1" />
      <dependency id="runtime.win7-x86.runtime.native.System.Data.SqlClient.sni" version="4.0.1" />
```

We need to understand if this is intentional or if folks were planning to build new versions of this component from the 1.1 codebase (presumably TFS?).

/cc @ianhays @saurabh500

**Comment by saurabh500** [comment]
@ericstj Where are you looking at these dependencies? 
This is not intentional. The SqlClient package should depend on new version sni built from TFS.

**Comment by ericstj** [comment]
It looks like @dagood made this happen in https://github.com/dotnet/corefx/pull/11814, but I want to make sure that it's intentional that we're shipping the 1.0 RTM binaries for the 1.1 release.  If I look at @dagoods PRs for rel and master it seems like its intentionally doing this: in master it pulls the latest build from TFS but for release/1.1.0 its pulling the old 1.0 packages.

**Comment by ericstj** [comment]
I'm seeing something similar for System.IO.Compression.  For the AOT packages they are all hardcoded to the RTM version.

**Comment by ericstj** [comment]
The following packages are all the ones that are not being rebuilt with 1.1 binaries.
- runtime.win10-x64-aot.runtime.native.System.IO.Compression
- runtime.win10-arm-aot.runtime.native.System.IO.Compression
- runtime.win10-x86-aot.runtime.native.System.IO.Compression
- runtime.win7-x64.runtime.native.System.Data.SqlClient.sni
- runtime.win7-x86.runtime.native.System.Data.SqlClient.sni

I've also noticed that we're inconsistent with how we're building these packages.  It looks like for IO.Compression we exclude the packages that build in TFS from the builds file, but for SNI those packages are in the builds file, but happen to be skipped because the binary is never built.

**Comment by dagood** [comment]
The 1.0.0 source for the release/1.1.0 branch is intentional: we don't have 1.1.0 TFS builds at all, so we're continuing to ship using 1.0.0 RTM TFS packages.
/cc @gkhanna79 @weshaggard

**Comment by ericstj** [comment]
Ok, I think I understand everything here now.  I'll need to remove the runtime.win10-arm64.runtime.native.System.Data.SqlClient.sni dependency then since that would prevent us from making this package stable.

**Comment by dagood** [comment]
Hmm, we'd decided to let it stay in an email thread about https://github.com/dotnet/corefx/issues/11805, but being unable to make the package stable sounds like new info.

**Comment by ericstj** [comment]
Yeah, when discussing before I thought folks were only looking at runtime.json referenced packages.  Since this one is a direct package reference, not in runtime.json, nuget would block stable -> unstable.

**Comment by saurabh500** [comment]
We need to open source SNI to take care of these issues.
@corivera is already working on it. 

We have identified an internal build of SNI package which can be shipped as part of 1.1 
Working with @weshaggard and @gkhanna79 over emails to coordinate the effort for shipping SNI from TFS for 1.1

**Comment by karelz** [comment]
Fixed by #13027

**Comment by dagoods** [comment]
folks: please be careful with @dagood vs. @dagoods
